### ISSUE TITLE
tagged posts results "400 Bad Request" even no reason given, while stories and highlights are downloading normally.

### ISSUE THREAD

**Issue by estatistics** [instaloader/instaloader#2418]
**Describe the bug**
downloading TAGGED posts are impossible, they arent downloading while stories and highlights are downloading.
What is happening? In many other profiles same error, or it let me to download some and then it stops with some error.

**To Reproduce**
using
`python3 /mnt/disk/__configs/615_import_firefox_session.py  -c /mnt/diskfirefox/default/cookies.sqlite`
to load cookies and then
`instaloader --login=USER --fast-update    --user-agent "Mozilla/5.0.........." --abort-on=302,400,401,429,403  --stories  --tagged --highlights Lauren.horner/`

**Expected behavior**
to download tagged posts

**Error messages and tracebacks**
Loaded session from /home/........
Logged in as .......
[1/1] Downloading profile lauren.horner
lauren.horner/2024-08-22_01-50-12_UTC_profile_pic.jpg already exists
Retrieving tagged posts for profile lauren.horner.
Resuming from lauren.horner/:tagged/iterator_Wha2sV92.json.xz.
[ 1]
Saved resume information to lauren.horner/:tagged/iterator_Wha2sV92.json.xz.

`Download aborted: Query to https://www.instagram.com/graphql/query responded with "400 Bad Request": {"message":"feedback_required","spam":true,"feedback_title":"Try Again Later","feedback_message":"We limit how often you can do certain things on Instagram to protect our community. Tell us if you think we made a mistake.","feedback_url":"","feedback_action":"report_problem","feedback_appeal_label":"Let us know","restriction_enrollment_data":"","status":"fail"}.`

without tagged posts...

```
Loaded session from .....
Logged in as .......
[1/1] Downloading profile lauren.horner
lauren.horner/2024-08-22_01-50-12_UTC_profile_pic.jpg already exists
Retrieving highlights "San Diego" from profile lauren.horner
lauren.horner/San Diego/2024-10-01_22-28-08_UTC_cover.jpg already exists
[  1/ 25] lauren.horner/San Diego/2024-10-02_23-45-05_UTC.jpg exists json 
Retrieving highlights "North Carolina" from profile lauren.horner
lauren.horner/North Carolina/2018-07-16_19-59-48_UTC_cover.jpg already exists
[  1/ 26] Warning: Unable to fetch video from graphql of <StoryItem 2585982453185443758>: list index out of range
lauren.horner/North Carolina/2021-05-31_20-32-30_UTC.mp4 lauren.horner/North Carolina/2021-05-31_20-32-30_UTC.jpg exists json 
Retrieving highlights "NYC" from profile lauren.horner
lauren.horner/NYC/2023-06-16_22-45-42_UTC_cover.jpg already exists
[  1/ 19] lauren.horner/NYC/2024-08-22_18-31-40_UTC.jpg exists json 
Retrieving highlights "Jax" from profile lauren.horner
lauren.horner/Jax/2024-07-12_00-27-10_UTC_cover.jpg already exists
[  1/ 27] Loaded session from /home/elias/.config/instaloader/session-fotomelodios.

[1/1] Downloading profile lauren.horner
lauren.horner/2024-08-22_01-50-12_UTC_profile_pic.jpg already exists
Retrieving highlights "San Diego" from profile lauren.horner
lauren.horner/San Diego/2024-10-01_22-28-08_UTC_cover.jpg already exists
[  1/ 25] lauren.horner/San Diego/2024-10-02_23-45-05_UTC.jpg exists json 
Retrieving highlights "North Carolina" from profile lauren.horner
lauren.horner/North Carolina/2018-07-16_19-59-48_UTC_cover.jpg already exists
[  1/ 26] Warning: Unable to fetch video from graphql of <StoryItem 2585982453185443758>: list index out of range
lauren.horner/North Carolina/2021-05-31_20-32-30_UTC.mp4 lauren.horner/North Carolina/2021-05-31_20-32-30_UTC.jpg exists json 
Retrieving highlights "NYC" from profile lauren.horner
lauren.horner/NYC/2023-06-16_22-45-42_UTC_cover.jpg already exists
....
Downloading stories
Saved session to /......
```

**Instaloader version**
`instaloader --version` output.
4.13

**Additional context**
Note that happens even days pass to download instagram tagged posts.

**Comment by aandergr** [comment]
Thank you for your bug report, which now aligns better with our template, and I appreciate the effort you've put into it.

However, I need to address the attitude displayed in your previous issue, which was, in no way, justified. Our bug reporting guidelines are not arbitrary; they are essential for ensuring that our small team of volunteers can effectively manage the hundreds of issues we receive each year. When users do not follow these guidelines, it not only creates unnecessary delays but also wastes valuable time for both parties.

The reporting bugs guide is well-visible when creating a bug report. In the bug report form, not only is the bug report template presented to you, making it straightforward to fill in the required details, but the guide is also linked directly there. You can find it here: [Reporting Bugs Guide](https://instaloader.github.io/contributing.html#reporting-bugs).

According to the guide, you should "briefly check whether the bug has already been reported." In this case, the bug you reported—indicated by the “400 Bad Request” error—has already been documented in an existing issue (#2384) with the same title. Checking that report would have not only saved us from discussing this issue multiple times, but it would have also directed you to a workaround that was shared very early on.

Additionally, the guide specifies that you should "ensure you use the latest version of Instaloader." This is critical because many issues arise from Instagram’s undocumented and frequently changing interface, which we address in our regular updates. You reported using version 4.13, while the most recent version is 4.13.2 (or 4.14rc2). Unfortunately, this information was missing from your previous bug report, which made it difficult for us to understand the context. The release notes for version 4.13.2 explicitly state, "Fix 400 Bad Request when obtaining full Post metadata," referencing the original “400 Bad Request” (#2384) bug report.

Upon testing, I can confirm that the reported issue is indeed fixed in version 4.13.2. Therefore, disregarding the well-communicated and necessary guidelines not only led to confusion but also wasted both your time and the time of our volunteer team.

In summary, your previous attitude was unwarranted, given the clear reporting guidelines that were not adhered to on multiple fronts. I encourage you to update to the latest version and check the existing bug reports before opening new ones in the future. If the problem persists after updating, feel free to reopen this report with any additional information.

Thank you for your understanding and cooperation.

**Comment by estatistics** [comment]
i will may ask sorry for my behavior if that led to upset you. I aftermath, thinking, what went wrong, i can think that "not planned" is a wrong terminology. I am in many forums and githubs subscribed. Usually, when there is missing information they say "missing information", and not tag it as "not planned".  So, the terminology i think is confusing.
